config/environments/development.rb

```diff
@@ -15,6 +15,8 @@
   # Show full error reports.
   config.consider_all_requests_local = true
 
+  config.jwt_auth_secret = '123'
```

Comment:
It might be worth separating this change out into it's own commit as it's not entirely related to the rest of the changes, and feels like something that could do with it's own commit message to explain why we had to do it.